ext/net/lib.deno_net.d.ts

```diff
@@ -546,4 +546,203 @@ declare namespace Deno {
    * @category Network
    */
   export function shutdown(rid: number): Promise<void>;
+
+  /** @category Network */
+  export interface QuicTransportOptions {
+    /** Period of inactivity before sending a keep-alive packet. Keep-alive
+     * packets prevent an inactive but otherwise healthy connection from timing
+     * out. Only one side of any given connection needs keep-alive enabled for
+     * the connection to be preserved. By default keep-alive is not enabled.
+     */
+    keepAliveInterval?: number;
+    /** Maximum duration of inactivity to accept before timing out the
+     * connection. The true idle timeout is the minimum of this and the peer’s
+     * own max idle timeout. By default idle timeouts are not enabled.
+     */
+    maxIdleTimeout?: number;
+    /** Maximum number of incoming bidirectional streams that may be open
+     * concurrently.
+     * Default: 100
+     */
+    maxConcurrentBidirectionalStreams?: number;
+    /** Maximum number of incoming unidirectional streams that may be open
+     * concurrently.
+     * Default: 100
+     */
+    maxConcurrentUnidirectionalStreams?: number;
+  }
+
+  /** @category Network */
+  export interface ListenQuicOptions extends QuicTransportOptions {
+    /** The port to connect to. */
+    port: number;
+    /** A literal IP address or host name that can be resolved to an IP address. */
```

Comment:
```suggestion
    /** A literal IP address or host name that can be resolved to an IP address.
    *
    * @default {"0.0.0.0"}
    */
```
Should we also add the same note that [`Deno.ServeOptions.hostname`](https://docs.deno.com/api/deno/~/Deno.ServeOptions.hostname) has?